package.json

```
@@ -13,13 +13,14 @@
     "next": "^14.2.12",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "tinacms": "^2.2.4",
+    "tinacms": "^2.2.9",
     "typescript": "5.5.2"
   },
   "devDependencies": {
-    "@tinacms/cli": "^1.6.4",
+    "@tinacms/cli": "^1.6.9",
     "@types/node": "^22.5.5",
     "eslint": "^8.57.1",
     "eslint-config-next": "^14.2.12"
-  }
+  },
+  "packageManager": "pnpm@9.11.0+sha512.0a203ffaed5a3f63242cd064c8fb5892366c103e328079318f78062f24ea8c9d50bc6a47aa3567cabefd824d170e78fa2745ed1f16b132e16436146b7688f19b"
```

```suggestion
  }
```

Remove package manager